### ISSUE TITLE
Cannot reproduce the results in terms of your provided best_vbpr.npy weights

### ISSUE THREAD

**Issue by axhiao** [duxy-me/AMR#3]
Hi, based on your weights located in the weights folder, I can only produce results close to VBPR in `table 2` of your paper.  But for AMR results, I cannot touch them. I have also tuned a lot about hyper-parameters. It still didn't work. Do you have any suggestions for me? Thanks!

**Comment by duxy-me** [comment]
Hi Minglei,
&nbsp; &nbsp; I cannot remember the details much sure. I could only say that, we just fine-tune the pretrained VBPR in a small learning rate for a few epochs.&nbsp;You may try different regularizations also.&nbsp;





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "duxy-me/AMR"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月26日(星期四) 凌晨0:21
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[duxy-me/AMR] Cannot reproduce the results in terms of your provided best_vbpr.npy weights (#3)





 
Hi, based on your weights located in the weights folder, I can only produce results close to VBPR in table 2 of your paper.  But for AMR results, I cannot touch them. I have also tuned a lot about hyper-parameters. It still didn't work. Do you have any suggestions for me? Thanks!
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
